rocAL/rocAL/CMakeLists.txt

```diff
@@ -188,6 +188,7 @@ if(${BUILD_ROCAL})
                 ./include/readers/image/
                 ./include/readers/video/
                 ./include/pipeline/
+                ./../../thirdparty/coco_RLE/include/
```

Comment:
Is this a cloned repo or just a single cpp file? If its a repo, better to add it as a dependency in Dockerfile and figure out how to import the RLE structs from there since they removed all 3rd party libs from rocAL and added them as dependencies

Reply:
Sure Sundar. We can discuss on it